ui/appui-react/src/appui-react/icons/AssetIsolate.tsx

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module AssetIsolate
+ */
+
+import * as React from "react";
+export function AssetIsolate() {
```

Comment:
Let's tag icons as `@internal`.

```suggestion
/** @internal */
export function AssetIsolate() {
```

Reply:
Added tags to every icon